<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input</class>
 <widget class="QWidget" name="input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>501</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="catogory">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>种类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>金属材料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>化工材料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>机电五金</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>建筑材料</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>车辆配件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水产品</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>房地产</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>人才市场</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="EI">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>进货出货</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>进货</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>出货</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>商品名字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="size">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>大</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="place_2">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>产地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="place">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>交易地点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="number">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="price">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="apartmentAddr">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>交易单位地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contactName">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>联系人姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contactTel">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>联系人电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
